app/components/events/event-import-section.js

```diff
@@ -26,9 +28,62 @@ export default Component.extend(FormMixin, {
       }
     };
   },
+  importTask(url) {
+    let _this = this;
+    this.get('loader')
+      .load(url)
+      .then(data => {
+        if (data.state !== 'SUCCESS') {
+          $('#import_status').html(`<b>Status:</b> ${  data.state}`);
+          setTimeout(function() {
+            _this.importTask(url);
+          }, 3000);
+        } else {
+          $('#import_status').html(`<b>Status:</b> ${  data.state}`);
+          document.location = `/events/${  data.result.id}`;
+        }
+      })
+      .catch(e => {
+        $('#import_status').text('');
+        $('#btnImportEvent').prop('disabled', false);
+        $('#import_file').prop('disabled', false);
+        $('#import_error').text(e.message);
+      });
+  },
   actions: {
     submit() {
+      let _this = this;
       this.onValid(() => {
+        var data = new FormData();
+        var endpoint = 'import/json';
+        $.each($('#file')[0].files, function(i, file) {
+          var ext = file.name.split('.');
+          ext = ext[ext.length - 1].toLowerCase();
+          if (ext === 'xml') {
+            endpoint = 'import/pentabarf';
+          } else if (ext === 'ical') {
+            endpoint = 'import/ical';
+          }
+          data.append('file', file);
+
+          $('#import_status').text('Uploading file.. Please don\'t close this window');
```

Comment:
Hey, check out the code for how the export of events his handled, we don't use JQuery selectors directly in the codebase, thank you!